src/components/Post/GovernanceSideBar/Modal/VoteData/VoteList.tsx

```diff
@@ -16,7 +16,7 @@ import nextApiClientFetch from '~src/util/nextApiClientFetch';
 import { network as AllNetworks } from '~src/global/networkConstants';
 import styled from 'styled-components';
 import VoterRow from './VoterRow';
-import ExpandIcon from '~assets/icons/expand-small-icon.svg';
+import ExpandIcon from '~assets/icons/expand-small-icon2.svg';
```

Comment:
remove this icon

Reply:
<ExpandIcon
											className={`text-xl text-bodyBlue dark:text-[#909090] ${
												heading === EHeading.PROPOSAL && !!sortByPostIndex ? 'ml-1 rotate-180 cursor-pointer' : 'ml-1 cursor-pointer'
											}`}
										/> use this instead